### PR TITLE
PIM-4205: fix locale specific

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | ?
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | N
| Fixed tickets        | PIM-4205
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: features/product/edit_locale_specific_value.feature [hunk 1]
```diff
@@ -12,13 +12,13 @@ Feature: Edit a locale specific value
     And I am logged in as "Mary"
     And I am on the "tshirt" product page
 
-  @skip-pef @jira https://akeneo.atlassian.net/browse/PIM-3298
+  @jira https://akeneo.atlassian.net/browse/PIM-3298
   Scenario: Display the custom tax on the available locale
     And I switch the locale to "de_DE"
-    And I visit the "Internal" group
+    And I visit the "Intern" group
```
Comment (jjanvier) [inline features/product/edit_locale_specific_value.feature]: intern ?
like `stagiaire` ??

**Comment by jmleroux** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM if behats are ok

File: features/product/edit_locale_specific_value.feature (hunk 1 above)
Comment (sumbobyboys) [inline features/product/edit_locale_specific_value.feature]: German :)

**Pull request merged**